src/polyfills/index.js

```diff
@@ -1,11 +0,0 @@
-import { enableES5, setAutoFreeze } from "immer";
-import "promise-polyfill/src/polyfill";
-import "whatwg-fetch";
-import "./d3-request.js";
-import "./innersvg.js";
-import "./isnan.js";
-
-enableES5();
-
-// d3 mutates state in many places, which is why we have to turn this off.
-setAutoFreeze(false);
```

Comment:
Are you sure we no longer need to disable auto-freezing?

Reply:
no idea 🤷 but it didn't break anything so not even sure what it was going

Reply:
I added it back to the `src/app.js` file since thats the only place where immer is used and keeps everything in one place (and nothing is breaking with or without it 🤷 )

Reply:
https://immerjs.github.io/immer/freezing/